network-api/networkapi/fellows/urls.py

```diff
@@ -1,13 +1,14 @@
 from django.conf.urls import url
 from networkapi.fellows import views
+from django.views.generic.base import RedirectView
 
 urlpatterns = [
     url(r'^$', views.fellows_home, name='fellowships-home'),
     url(r'^directory/$',
         views.fellows_directory,
         name='fellowships-directory'),
     url(r'^support/$', views.fellows_support, name='fellowships-support'),
-    url(r'^apply/$', views.fellows_apply, name='fellowships-apply'),
```

Comment:
@alanmoo R? I removed this line completely. Will set up redirect once this PR has been pushed to prod.